### DISCUSSION TITLE
How to map <C-Tab>/<C-S-Tab>

### DISCUSSION THREAD

**Discussion by c02y** [LunarVim/LunarVim#2449] [Q&A] [answered]
I want to bind `<C-Tab>/<C-S-Tab>` to switch tabs like browsers
```
lvim.keys.normal_mode["C-<TAB>"] = ":BufferLineCycleNext<CR>"
lvim.keys.normal_mode["C-S-<Tab>"] = ":BufferLineCyclePrev<CR>"

```

But `["<Tab>"]` works fine, I cannot make `["C-<Tab>"]` or `["C-<Tab>"] ` or `["<C-<Tab>>"] ` work

So, how to map `Ctrl+Tab`?

**Comment by abzcoding** [comment, marked as answer]
https://stackoverflow.com/a/1646910/3147164

**Reply by c02y** [reply]
Yeah, got this issue a long time ago, just forgot it.
